### PR TITLE
pump client: add initial log function

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What problem does this PR solve? 
need print pump client's log in tidb, use tidb's log config.
https://internal.pingcap.net/jira/browse/TOOL-764

### What is changed and how it works?
use tidb's log config function

### Check List 

Tests 

 - Manual test

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
@GregoryIan @july2993 PTAL

File: tidb-binlog/pump_client/client.go [hunk 1]
```diff
@@ -49,7 +50,7 @@ const (
 )
 
 var (
-	// Logger is ...
+	// Logger is ..., obsolete now.
 	Logger = log.New()
```
Comment (july2993) [inline tidb-binlog/pump_client/client.go:54]: just delete it if no use

**Review by july2993 — COMMENTED** [review]
rest LGTM

File: tidb-binlog/pump_client/client.go (hunk 1 above)
Comment (WangXiangUSTC) [reply tidb-binlog/pump_client/client.go:54]: need update tidb's code, and then I will clean all obsolete fields

**Comment by IANTHEREAL** [comment]
LGTM

**Review by july2993 — APPROVED** [review]
LGTM

**Pull request merged**